### ISSUE TITLE
Why does `/connect/authorize` use `/Identity/Account/Login` instead of `/Pages/Account/Login`?

### ISSUE THREAD

**Issue by khteh** [DuendeSoftware/Support#525]
**Which version of Duende BFF are you using?**
Community
**Which version of .NET are you using?**
7.0.102
**Describe the bug**
Specifically, where does this page come from? I don't have this page in the application.

![image](https://user-images.githubusercontent.com/3871483/220019683-2557390e-a1e7-4941-af34-93234dd93999.png)

**To Reproduce**

Steps to reproduce the behavior.

**Expected behavior**
Use the provided `/Pages/Account/Login/Index` for login page.

**Log output/exception with stacktrace**

```
data
```

**Additional context**

**Comment by josephdecock** [comment]
Not sure why this was opened and then closed, but I assume you must have found the LoginUrl configuration option. In case someone stumbles onto this thread someday, see [here](https://docs.duendesoftware.com/identityserver/v6/reference/options/#userinteraction).

**Comment by khteh** [comment]
No, I didn't find the `LoginUrl` configuration. It was due to https://github.com/DuendeSoftware/Support/issues/521 that I removed the scaffolded Identity pages which still ghosted around. Now with this config available, I can safely remove the scaffolded pages and use the custom ones in a different folder structure instead of having to replacing the scoffolded artifacts at `/Areas/...`
